assets-src/js/multiselect-module.js

```diff
@@ -407,7 +417,13 @@ MultiselectButtons.prototype.onInputKeyDown = function (event) {
             event.preventDefault();
             const nextFilteredIndex = getUpdatedIndex(activeFilteredIndex, max, action);
             const nextRealIndex = this.options.indexOf(this.filteredOptions[nextFilteredIndex]);
-            return this.onOptionChange(nextRealIndex);
+         case MenuActions.Space:
+            if (this.activeIndex) {
+                event.preventDefault();
+                return this.onOptionClick(this.activeIndex);
+            }
+            return;
```

Comment:
Which return should be kept?

Reply:
fixed in https://github.com/w3c/w3c-website-templates-bundle/pull/58/commits/f4740c33656a09209121b8564cf33ca55172ae0d